doc/platform_api/CMIS_Diagnostic_Monitoring_Overview_in_SONiC.md

```diff
@@ -2427,10 +2450,10 @@ When the transceiver is inserted back, the `TRANSCEIVER_DOM_FLAG_CHANGE_COUNT`,
 
 #### 5.2.4 Diagnostic Information Last Update Timestamp and Interval Period by `DomInfoUpdateTask`
 
-All the diagnostic tables (except for the metadata tables storing change count and last set/clear time) contain the `table_last_update_time` field to capture the last update timestamp.
+All the diagnostic tables (except for the metadata tables storing change count and last set/clear time) contain the `last_update_time` field to capture the last update timestamp.
 Specifically, the `TRANSCEIVER_STATUS` table contains the `diagnostics_update_interval` field to capture the interval period at which the diagnostic information is updated by the `DomInfoUpdateTask` thread for a port. This field is not present in the other diagnostic tables since the diagnostic information is updated for all ports in a sequential manner.
 
-1. **`table_last_update_time`**:
+1. **`last_update_time`**:
```

Comment:
@mihirpat1 are we planning to make `diagnostics_update_interval` visible to user for info?

Reply:
@prgeor Yes, we plan to display in the `Update interval:` section of the CLI output